packages/snap-toolbox/src/charsParams/charsParams.ts

```diff
@@ -15,9 +15,9 @@ export function charsParams(params: ParameterObject): number {
 					return length + keyLength + 1 + ('' + val).length;
```

Comment:
Because `value` here can be an array of anything now (due to new "unknown" type) we will likely get some inaccurate numbers for our count. I think the `'' + val` ought to take care of all potential array types, we may want to add some testing to ensure that we are not throwing any errors when weird types are passed into the function.